Library/Source/RuntimeImpl.h

```diff
@@ -5,19 +5,31 @@
 #include <arcana/threading/dispatcher.h>
 #include <arcana/threading/task.h>
 
+namespace Napi
+{
+    class Env;
+}
+
 namespace babylon
 {
     class Env;
-    class NativeEngine;
+    class NativeWindow;
 
-    class RuntimeImpl
+    class RuntimeImpl final
     {
+        static constexpr auto JS_NATIVE_NAME = "_native";
+        static constexpr auto JS_RUNTIME_NAME = "runtime";
+        static constexpr auto JS_WINDOW_NAME = "window";
+        static constexpr auto JS_ENGINE_CONSTRUCTOR_NAME = "Engine";
```

Comment:
nit: I don't like having fields not under a `<access>:` section. Do these even need to be in the header at all?